telegram/chat.py

```diff
@@ -53,11 +53,11 @@ class Chat(TelegramObject):
     def __init__(self,
                  id,
                  type,
-                 title='',
-                 username='',
-                 first_name='',
-                 last_name='',
-                 all_members_are_admins=False,
+                 title=None,
+                 username=None,
+                 first_name=None,
+                 last_name=None,
+                 all_members_are_admins=None,
```

Comment:
I'm about 90% sure they changed their API... so should we change it, or like keep both, or?